opentracing-util/src/main/java/io/opentracing/util/GlobalTracer.java

```diff
@@ -131,6 +131,35 @@ public static synchronized boolean registerIfAbsent(final Callable<Tracer> provi
         return false;
     }
 
+    /**
+     * Register a {@link Tracer} to back the behaviour of the {@link #get()}.
+     * <p>
+     * Registration is a one-time operation. Once a tracer has been registered, all attempts at re-registering
+     * will return {@code false}. Use {@link #registerIfAbsent(Callable)} for use cases when the tracer
```

Comment:
> Use `registerIfAbsent(Callable)` for use cases where the tracer can be instantiated multiple times.

I still don't understand what you mean by this comment.

Reply:
I think we mean the same same thing but maybe a wording can be improved.

Does this look good to you? 

> Use {@link #registerIfAbsent(Callable)} for lazy initiation to avoid multiple instantiations of tracer.

Reply:
Yes, better.

Reply:
Thanks!